horton/gbasis/gobasis.py

```diff
@@ -230,6 +230,8 @@ def load(self):
         """Load the basis set from file."""
```

Comment:
Could you also add a line that just returns if basis_atom_maps is already present? This is related to comment below, i.e. to make sure that loading twice is harmless.